File: __tests__/fixtures/output/java/okhttp/nested.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")
  .get()
  .build();

Response response = client.newCall(request).execute();

File: __tests__/__fixtures__/output/kotlin/okhttp/nested.kt
val client = OkHttpClient()

val request = Request.Builder()
  .url("https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")
  .get()
  .build()

val response = client.newCall(request).execute()
